### PR TITLE
Fix mobile reschedule actions

### PR DESCRIPTION
**Pull request by hariombalhara**
## What does this PR do?
On mobile reschedule, actions aren't working. Only the "Reschedule" option is coming which has been divided into two parts now "Edit Booking", "Request reschedule".  This is the reason, **the action itself isn't working.**

Clock Icon has been used for both actions for now. They should be changed to appropriately reflect the two actions.

Earlier:
Screenshot 2022-04-17 at 5 18 50 PM

Now:
Screenshot 2022-04-17 at 5 18 16 PM

Also, "Send reschedule request" text was too big which was breaking the button UI. It has been changed to "Request reschedule"

Earlier:
Screenshot 2022-04-17 at 5 21 22 PM

Now:
Screenshot 2022-04-17 at 5 24 05 PM

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How should this be tested?

- [ ] Test A
- [ ] Test B

## Checklist

- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by alannnc — APPROVED** [review]

File: apps/web/public/static/locales/en/common.json [hunk 1]
```diff
@@ -760,7 +760,7 @@
   "reschedule_request_sent": "Reschedule request sent",
   "reschedule_modal_description": "This will cancel the scheduled meeting, notify the scheduler and ask them to pick a new time.",
   "reason_for_reschedule_request": "Reason for reschedule request",
-  "send_reschedule_request": "Send reschedule request",
+  "send_reschedule_request": "Request reschedule ",
```
Comment (zomars) [inline apps/web/public/static/locales/en/common.json:763]: Why was this changed @hariombalhara @alannnc

**Comment by hariombalhara** [comment]
@zomars  "Send reschedule request" text was too big which was breaking the button UI. It has been changed to "Request reschedule"

Also, see the screenshot in the description
